src/lib/logger/index.js

```diff
@@ -4,6 +4,7 @@ import {
 	transports,
 } from 'winston'
 
+import utils from '%src/lib/utils/env'
```

Comment:
In other places we have used named imports for utils -- e.g.

```
import {
  isProductionEnv,
} from '%src/lib/utils/env'
```

Might be good to be consistent there, though also appreciate if maybe that's not good practice somehow.

That said even if you don't do the named import thing, `utils` feels a bit generic as opposed to `env` or `envUtils`.

Reply:
By the way, I *think* the logic for explicitly pulling the util methods being invoked was it was explicit and as a habit it helps with tree shaking.

Reply:
Agree! This was how I tried to do it at first and for some reason it wasn't playing nice with the default export I was doing, which in turn was based on a misreading of the `import/prefer-default-export` warning. This is all solved now.